app/web/src/components/EditorMenu/menuConfig.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { useRender } from 'src/components/IdeWrapper/useRender'
+import { makeStlDownloadHandler, PullTitleFromFirstLine } from './helpers'
+import { useSaveCode } from 'src/components/IdeWrapper/useSaveCode'
+import { DropdownItem } from './Dropdowns'
+
+export function cmdOrCtrl() {
+    return /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform) ? '⌘' : 'Ctrl'
+}
+
+const fileMenuConfig = {
```

Comment:
if you make this:
```suggestion
const fileMenuConfig: EditorMenuConfig = {
```
Thdn the linter (assuming you've got ts setup with your editor) should tell you if your obeying your own types for this menu, and I think you're not since `component` doesn't exist on `EditorMenuItemConfig`.

Reply:
Yup it yelled at me now!